cosmos/carbon.json

```diff
@@ -262,19 +262,19 @@
       "coinDecimals": 6,
       "coinGeckoId": "cosmos",
       "gasPriceStep": {
-        "low": 0.007692307692307693,
-        "average": 0.009615384615384616,
-        "high": 0.011538461538461539
+        "low": 0.003333333333,
+        "average": 0.003333333333,
+        "high": 0.003333333333
       }
     },
     {
       "coinDenom": "stATOM",
       "coinMinimalDenom": "ibc/B7864B03E1B9FD4F049243E92ABD691586F682137037A9F3FCA5222815620B3C",
       "coinDecimals": 6,
       "gasPriceStep": {
-        "low": 0.007692307692307693,
-        "average": 0.009615384615384616,
-        "high": 0.011538461538461539
+        "low": 0.003333333333,
+        "average": 0.003333333333,
+        "high": 0.003333333333
```

Comment:
Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas.